src/js/components/Avatar/__tests__/Avatar-test.js

```diff
@@ -101,4 +101,19 @@ describe('Avatar', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('text renders with different sizes', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Avatar background="dark-2" textSize="small">
```

Comment:
I think we should transfer the test to testing the theme value size and its corresponding textSize. 
One request, if you can separate this test to a separate PR, so we can make sure no changes in the snapshot are being made with this PR it will be fantastic.

Reply:
Should I move docs update aswell? As it updates README snapshot.

Reply:
Yep.

Reply:
No problem. I'll let you know when those things are done. I'm just not sure if I understood the 

> And on this PR, you can just add the reference to avatar.text.size once it exists.

part.Are you talking about the docs edit or tests? Sorry for that.
Do you mind elaborating a little more? Thanks.

Reply:
Yes, sorry for not being clear, and thanks for your patience. 
Just keep the tests on this PR, no need for the extra complexity.